src/UI/Components/Diagram/styles.ts

```diff
@@ -0,0 +1,257 @@
+import styled from "styled-components";
+
+export const CanvasWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: relative;
+  background: #f3f7f6;
+  margin: 0;
+  font-family: sans-serif;
+  overflow: hidden;
+  .canvas {
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    position: absolute;
+    background: #f3f7f6;
+
+    .joint-element {
+      filter: drop-shadow(0.1rem 0.1rem 0.15rem rgba(0, 0, 0, 0.2));
+    }
+    .joint-paper-background {
+      background: #f3f7f6;
+    }
+
+    .source-arrowhead,
+    .target-arrowhead {
+      fill: #a0a0a0;
+      stroke-width: 1;
+    }
+  }
+
+  /* 
+    ***  ui.Dialog ***
+  */
+  .joint-dialog.joint-theme-default {
+    .body {
+      padding: 0;
+      max-height: 500px;
+      overflow-y: scroll;
+    }
+
+    .fg {
+      border: none;
+      border-radius: 0px;
+      filter: drop-shadow(0.1rem 0.1rem 0.15rem rgba(0, 0, 0, 0.2));
+    }
+
+    .titlebar {
+      padding: 16px;
+      border-radius: 0px;
+      color: #3d3d3d;
+      background: #ffffff;
+      border: none;
+      border-bottom: 1px solid #dddddd;
+      font-size: 20px;
+
+      .titletab {
+        height: 5px;
+        position: absolute;
+        top: -5px;
+        left: 0;
+        width: 100%;
+      }
+    }
+
+    .controls {
+      border: none;
+      border-top: 1px solid #dddddd;
+
+      .control-button {
+        color: #303030;
+        border: 1px solid #cccccc;
+        background: #ffffff;
+        border-radius: 0px;
+        min-width: 82px;
+
+        &:hover {
+          border: 1px solid #cccccc;
+          background: #cccccc;
+          opacity: 0.8;
+          transition: 0.1s linear;
+        }
+
+        // Submit button
+        &.left {
+          color: #f8f8ff;
+          background: blue;
+          border: 1px solid blue;
+        }
+      }
+    }
+  }
+
+  /* 
+    ***  ui.Inspector ***
```

Comment:
Multiline comments have this format : 
```
/**
 * some text
 * some text
 */
```

Reply:
changed to one-line as it gets to "starry" in my opinion :D